tests/integration/test_logtest/test_remove_old_sessions/test_remove_old_session_for_inactivity.py

```diff
@@ -42,7 +51,8 @@ def get_configuration(request):
 
 
 # Test
-def test_remove_old_session_for_inactivity(get_configuration, configure_environment,
+def test_remove_old_session_for_inactivity(get_local_internal_options, configure_local_internal_options,
+                                           get_configuration, configure_environment,
                                            restart_wazuh, connect_to_sockets_function):
```

Comment:
After talking with @snaow we have considered that new development or changes into existing tests should follow new repository standards about Wazuh daemons. For now, it is required to use fixtures to start only required daemons, and turn them off when the test is over. That means, that the default status of Wazuh would be off and each test should be responsible for daemons handling.
The reasons for this change are:

- Faster testing
- More encapsulated tests